### ISSUE TITLE
Error displaying widget: model not found

### ISSUE THREAD

**Issue by gqpj** [QuantStack/ipycytoscape#22]
Hello! 
I'm trying to run examples in JupyterLab 2.0.1:
```
pip install ipycytoscape
jupyter labextension install @jupyter-widgets/jupyterlab-manager
jupyter labextension install jupyter-cytoscape@0.1.2
```
I've got the error in every example: 
```
cytoscapeobj
Error displaying widget: model not found
```

What I'm doing wrong?

**Comment by gqpj** [comment]
@marimeireles I see you hard-working on your fork may be you already fixed this?

**Comment by jtpio** [comment]
@gqpj what is the output of `jupyter labextension list`?

**Comment by gqpj** [comment]
@jtpio 
```
$ jupyter labextension list                                                                                                                                                                                                            
JupyterLab v2.0.1
Known labextensions:
   app dir: /home/gqpj/miniconda3/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        jupyter-cytoscape v0.1.4  enabled  OK
```

**Comment by gqpj** [comment]
The installation:
```
$ jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-cytoscape                                                                                                                                                   
Building jupyterlab assets (build:prod:minimize)
\
$
```
Is it ok that the process ends `\` symbol?

**Comment by gqpj** [comment]
I've rebuilt with `--minimize=False`:
```
jupyter labextension install --minimize=False @jupyter-widgets/jupyterlab-manager jupyter-cytoscape
```
and it works now.

**Comment by jtpio** [comment]
Thanks @gqpj for posting your answer.

However running with `--minimize=False` shouldn't be required.

Also the list above suggests that you are using the base conda environment? Would you be able to try the commands (without the minimize flag) in a new environment?

**Comment by gqpj** [comment]
It works in fresh conda env without minimization. Thanks!
```
conda create --name jtest
conda activate jtest
conda install jupyterlab
pip install ipycytoscape
jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-cytoscape 
```

**Comment by abitofalchemy** [comment]
> ```
> jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-cytoscape 
> ```

did not work for me I still have the issue. 

here is my `jupyter labextension list`

```
JupyterLab v3.0.0
/Users/ag23268/opt/anaconda3/envs/jtest/share/jupyter/labextensions
        @jupyter-widgets/jupyterlab-manager v3.0.0 enabled OK (python, jupyterlab_widgets)

Other labextensions (built into JupyterLab)
   app dir: /Users/ag23268/opt/anaconda3/envs/jtest/share/jupyter/lab
        jupyter-cytoscape v1.0.4 enabled OK
```

**Comment by ianhi** [comment]
Hey @abitofalchemy your error is because ipycytoscape has not yet been updated for jupyterlab 3. There's an unfinished PR here https://github.com/QuantStack/ipycytoscape/pull/171. Likely this will get done sometime in the coming week - though if you want to champion that PR forward that would also be welcome of course :)

**Comment by marimeireles** [comment]
@abitofalchemy  yes, I think the PR is actually working, iirc. We just need to test it and merge it, in case you want to use it with ipycytoscape.
Either that or you fix the jupyter lab version to be 2.2.8, and should work =)
